Library/Homebrew/cask/lib/hbc/dsl.rb

```diff
@@ -206,16 +204,6 @@ def sha256(arg = nil)
       @sha256 ||= arg
     end
 
-    def license(arg = nil)
```

Comment:
Looks like this breaks the `brew cask list` call in `test/test_cask.rb` because `odeprecated` throws an error when running as `HOMEBREW_DEVELOPER`. In `test_cask.rb`, change `cmd("cask", "list")`, to `cmd("cask", "list", "--caskroom=#{HOMEBREW_PREFIX}/Caskroom")`